src/components/Home.jsx

```diff
@@ -0,0 +1,9 @@
+import Articles from "./Articles"
+
+export default function Home({setArticles}) {

```

Comment:
A prop being passed here but not on the side where it's being rendered. Is this prop doing anything?

Reply:
That prop is not currently doing anything for the moment. but i am intending on making the articles re-render when the Homepage link is selected. But i have not imported my setArticles State into my App.js as of yet.